README.md

```diff
@@ -135,6 +135,10 @@ Flags:
 When the limit set is less that the actual number of versions then only the latest release versions up to the limit will be converted. Older release versions with not be converted.
 If `--delete-v2-releases ` is set, thes older versions will remain in Helm v2 storage but will no longer be visible to Helm v2 commands like `helm list`. [Clean up](#clean-up-helm-v2-data)
 will remove them from storage.
+You can use below command to migrate all releases:
```

Comment:
I think this would be better placed in a new section called "Frequently Asked Questions". You could have a subsection called "Performing Helm v2 release migration in batch mode". The details could be similar to the following:

You can perform batch migration of releases using a command as follows:

```
`kubectl get [configmap|secret] -n <tiller_namespace> -l "OWNER=TILLER" | awk '{print $1}' | grep -v NAME | cut -d '.' -f1 | uniq | xargs -n1 helm3 2to3 convert`
```

An example of migrating releases which are stored as ConfigMaps in Tiller namespace `kube-system`:

```
kubectl get configmap -n kube-system -l "OWNER=TILLER" | awk '{print $1}' | grep -v NAME | cut -d '.' -f1 | uniq | xargs -n1 helm3 2to3 convert
```

Reply:
Looks good, done. Thanks for review @hickeyma